README.md

```diff
@@ -1,28 +1,45 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Ably reactjs-realtime-commenting tutorial
+
+- -> Step 16 (optional) – Add a profanity filter
+- [Step 15 – Displaying Comments](https://github.com/marsielko/tutorials/tree/d8ebeaca0a7a7c647e7b8df0c2abaada8ff63c32)
+- [Step 13 – Updating State With Newly Added Comment](https://github.com/marsielko/tutorials/tree/f3e0e74969c08ccbee54cd36c20d8ac557426ec9)
+- [Step 12 – Create App Component](https://github.com/marsielko/tutorials/tree/aadcf9f06d1e5bd69b0d4063d51cbc5b364c6e98)
+- [Step 11 – Create Comments Component](https://github.com/marsielko/tutorials/tree/2455b268dfdb806f4d0e8d72b97e2c0091d75436)
+- [Step 10 – Create Comment Component](https://github.com/marsielko/tutorials/tree/ba2260c37f5343606649b69d726678e74db9cd5c)
+- [Step 9 – Adding and Publishing Comment](https://github.com/marsielko/tutorials/tree/4689eef90b22bdfafedc1b60d18dbdeae128e7d7)
+- [Step 8 – Create CommentBox Component](https://github.com/marsielko/tutorials/tree/d85123d92cf55afd4debe3a6a7df0850e4d3fe53)
+- [Step 7 – Install Ably](https://github.com/marsielko/tutorials/tree/c2bcbbea6407e5ff9d18f8d65b9a112c6dbef574)
+- [Step 6 – Add Bulma CSS Framework](https://github.com/marsielko/tutorials/tree/603f0ee7f1035db89b025d5a61d543d2838e0a0f)
+- [Step 5 – Remove References To Unused Files](https://github.com/marsielko/tutorials/tree/dcb1a3a77126f2ab84611a1b883f1113c0d5e02f)
+- [Step 4 – Create Component Folder](https://github.com/marsielko/tutorials/tree/315e379ea668e44aa7348a6efea76d979a2eab7c)
+- [Step 3 – Delete Unused Files](https://github.com/marsielko/tutorials/tree/9db6c81a6dc24d4d8ee7008ce895e67e4eed3ef5)
+- [Step 2 – Create a React App](https://github.com/marsielko/tutorials/tree/c4815ffbb58a97be4d5848c37ab4acd11b63b21d)
+
 ## Available Scripts
 
 In the project directory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.<br>
+Runs the app in the development mode.<br />
```

Comment:
A double space will have the same effect as a <br/> here, and I think looks neater.

Reply:
(same for other brs)

Reply:
I agree with you. <br>s replaced with double spaces.